src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -1169,28 +1175,28 @@ void processZoneEquipSplitterInput(EnergyPlusData &state,
     static constexpr std::string_view RoutineName("processZoneEquipSplitterInput: "); // include trailing blank space
     auto &ip = state.dataInputProcessing->inputProcessor;
     std::string const zeqTypeName = ip->getAlphaFieldValue(objectFields, objectSchemaProps, "zone_equipment_object_type");
-    thisZeqSplitter.equipType = DataZoneEquipment::ZoneEquipType(getEnumValue(zoneEquipTypeNamesUC, zeqTypeName));
+    thisZeqSplitter.zoneEquipType = DataZoneEquipment::ZoneEquipType(getEnumValue(zoneEquipTypeNamesUC, zeqTypeName));
     // SpaceHVAC TODO: Copied this block from processZoneEquipmentInput section for ZoneHVAC:EquipmentList - seems this could be simplified
-    if (thisZeqSplitter.equipType == ZoneEquipType::Invalid) {
+    if (thisZeqSplitter.zoneEquipType == ZoneEquipType::Invalid) {
```

Comment:
Good, this is renamed :+1:

Reply:
Right, only the splitter references a specific piece of zone equipment, so `zoneEquipType` stores that.

Reply:
The only thing I see here is that zoneEquipType can't be invalid? coverage would show if this is true.

    SpaceHVAC:ZoneEquipmentSplitter,
     A3, \field Zone Equipment Object Type
       \required-field
       \type choice
       \key ZoneHVAC:TerminalUnit:VariableRefrigerantFlow
       \key ZoneHVAC:AirDistributionUnit
       \key ZoneHVAC:EnergyRecoveryVentilator
       \key ZoneHVAC:EvaporativeCoolerUnit
        \ ...

Reply:
Oh, but it *can* be invalid, because `ZoneEquipType` doesn't include all of the valid equipment types.
```
        if (zeqTypeName == "ZONEHVAC:LOWTEMPERATURERADIANT:CONSTANTFLOW" || zeqTypeName == "ZONEHVAC:LOWTEMPERATURERADIANT:ELECTRIC") {
            thisZeqSplitter.zoneEquipType = ZoneEquipType::LowTemperatureRadiant;
        } else if (zeqTypeName == "WATERHEATER:HEATPUMP:WRAPPEDCONDENSER") {
            thisZeqSplitter.zoneEquipType = DataZoneEquipment::ZoneEquipType::HeatPumpWaterHeater;
```
This was copied from the [zone equipment list code](https://github.com/NREL/EnergyPlus/blob/1c467dfa92b5e8cd2c8eb9b3a39ea0da92621d36/src/EnergyPlus/DataZoneEquipment.cc#L948-L957). Both of these can be cleaned up (someday) by extending the enum to include the missing equipment types.